src/app/layout/app-info/reminder-timeline/reminder-timeline.component.ts

```diff
@@ -0,0 +1,228 @@
+        renderTo: 'timeline',
+        margin: [0, 0, 0, 0]
+      },
+      tooltip: {
+        enabled: false
+      },
+      scrollbar: {
+        barBackgroundColor: '#dfdfdf',
+        barBorderRadius: 7,
+        barBorderWidth: 0,
+        buttonBackgroundColor: 'none',
+        buttonBorderWidth: 0,
+        buttonBorderRadius: 7,
+        trackBackgroundColor: 'none',
+        trackBorderWidth: 0,
+        trackBorderRadius: 8,
+        trackBorderColor: '#dfdfdf'
+      },
+      title: null,
+      xAxis: {
+        plotLines: [{
+          value: 1499433993000, // new Date().getTime(),
+          width: 2,
+          color: '#d0d6e2',
+          label: {
+            text: '',
+            useHTML: true,
+            verticalAlign: 'bottom',
+            class: 'sami'
+          }
+        }],
+        min: 1499425233000,
+        max: 1499443233000,
+        crosshair: false
+      },
+      yAxis: {
+        labels: {
+          enabled: false
+        },
+        title: {
+          text: null,
+        },
+        opposite: true,
+        gridLineWidth: 0,
+        min: 0,
+        max: 10
+      },
+      credits: {
+        enabled: false
+      },
+      exporting: {
+        enabled: false
+      },
+      navigator: {
+        enabled: false
+      },
+      series: /*this.timelineSeries()*/[{
+        data: [[1499382584000, 5], [1499389784000, 5]],
+        lineWidth: 10,
+        name: {
+          title: 'Confort',
+          icon: 'icon-bed'
+        }
+      }, {
+        data: [[1499396984000, 2], [1499407784000, 2]],
+        lineWidth: 10,
+        name: {
+          title: 'Eco',
+          icon: 'icon-leaf'
+        }
+      }, {
+        data: [[1499411384000, 8], [1499418584000, 8]],
+        lineWidth: 10,
+        name: {
+          title: 'Night',
+          icon: 'icon-moon'
+        }
+      }, {
+        data: [[1499425658000, 5], [1499432858000, 5]],
+        lineWidth: 10,
+        name: {
+          title: 'Game',
+          icon: 'icon-gamepad'
+        }
+      }, {
+        data: [[1499429258000, 8], [1499440058000, 8]],
+        lineWidth: 10,
+        name: {
+          title: 'Recreation',
+          icon: 'icon-keyboard'
+        }
+      }, {
+        data: [[1499436458000, 2], [1499447258000, 2]],
+        lineWidth: 10,
+        name: {
+          title: 'Visit',
+          icon: 'icon-stethoscope'
+        }
+      }, {
+        data: [[1499466393000, 5], [1499469993000, 5]],
+        lineWidth: 10,
+        name: {
+          title: 'Work',
+          icon: 'icon-briefcase'
+        }
+      }, {
+        data: [[1499473593000, 2], [1499484393000, 2]],
+        lineWidth: 10,
+        name: {
+          title: 'Shopping',
+          icon: 'icon-shopping-bag'
+        }
+      }, {
+        data: [[1499477193000, 5], [1499491593000, 5]],
+        lineWidth: 10,
+        name: {
+          title: 'Confort',
+          icon: 'icon-flash'
+        }
+      }]
+    };
+  }
+
```

Comment:
Please refer to the issue no [#42](https://github.com/smart-dashboard/angular-dashboard/issues/42)